src/web/docusaurus/docs/getting-started/environment-setup.md

```diff
@@ -240,6 +240,24 @@ Docker builds Telescope's dependencies at launch and keeps them on disk. In some
 
 :::
 
+### Seeding PostgresSQL
```

Comment:
I think this section should be part of the `Start telescope` section since the `pnpm db:init` script (not a command), won't work unless telescope, and specifically the db containers, are already running. Also, I think the env var in [this env file](https://github.com/Seneca-CDOT/telescope/blob/ec80669bb681cd76f9080299659541f46da70c4f/src/db/env.example#L7) needs to be present in the environment.

Also, make sure your instructions work before pushing your changes to this PR.